webdriver-spec.html

```diff
@@ -3946,6 +3946,9 @@
  <li><p>Let <var>centre point</var> be a pair of <var>rect</var>’s
   (<a>rectangle x coordinate</a> + <var>horizontal centre offset</var>)
   and (<a>rectangle y coordinate</a> + <var>vertical centre offset</var>)
+  
+ <li><p>If <a>command</a> is intercepted by an <var>element</var> with a higher 
+  z-index, an <a>element intercepting click</a> <a>error</a> is returned.
```

Comment:
Basically `elementsFromPoint` gets the paint ordering in which each element has been drawn. This seems like a more sound basis to use when determining if an element that is attempted clicked is actually clickable.

However, as @burg points out, whether an element is immediately clickable doesn’t say anything about whether the click bubbles down to an underlying element.

I think it is a conceptually good change to have a separate error for when another element is on top and a click won’t fulfil, but I’m unsure how we can effectively describe this.